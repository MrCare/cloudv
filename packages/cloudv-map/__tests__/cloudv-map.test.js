
import { mount } from '@vue/test-utils'
import cloudvMap from "../src/cloudv-map.vue";

describe("cloudvMap.test.js", () => {
  // 现在挂载组件，你便得到了这个包裹器
  const wrapper = mount(cloudvMap, {
    propsData: {
      message: 'World1'
    }
  })

  it('message includes "world"', () => {
    expect(wrapper.html()).toContain("World1");
  });

  it("has the expected html structure", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
