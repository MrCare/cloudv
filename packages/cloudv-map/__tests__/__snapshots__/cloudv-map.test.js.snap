// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.test.js has the expected html structure 1`] = `
<div message="World1">
    <p class="demo-box">
        cloudv-map
    </p>
</div>
`;

exports[`cloudvMap.test.js has the expected html structure 1`] = `
<div message="World1">
    <p class="demo-box">
        cloudv-map
    </p>
</div>
`;
